vast/integration/reference/server-zeek-conn-log/step_07.ref

```diff
@@ -16,6 +16,7 @@
 {"path":["index","num-cached-partitions"],"type":"number"}
 {"path":["index","num-unpersisted-partitions"],"type":"number"}
 {"path":["index","partitions","active",0,"id"],"type":"string"}
+{"path":["index","statistics","all-layouts","total"],"type":"number"}
```

Comment:
Imho the extra `all-layouts` key doesn't really add value, how about just:
```suggestion
{"path":["index","statistics","events"],"type":"number"}
```

Reply:
We could also switch the structure more generally, e.g.,

```json
"events": {
  "total": 123,
  "schema": {
    "suricata": {
      "alert": 1,
      "flow": 19
    },
    "zeek": {
      "conn": 100,
      "http": 3
    }
  }
}
```

Reply:
So the new output of `vast status --detailed` should be like:
```yaml
{
  ...
  "index": {
    ...
    "events": {
      "total": 25711
       schema": {
         "suricata": {
           "alert": 1,
           "flow": 19
       },      
       ...     
    },
    ...
  },
  ...
}
```

Reply:
Do we achieve that format by splitting the `name` in `stats.layouts` by dots?

https://github.com/tenzir/vast/blob/622a00dc8c5501c97bc8df8e5e9fdeb1deb6674f/libvast/src/system/index.cpp#L638

Reply:
It's just an idea for an improved UX. Sounds like a good thing to discuss in Slack quickly based on a list of possible outputs. We need to break here anyways given that we want to `s/schema/module` and `s/layout/schema`, and while we're at it we should also think about the ergonomics.